src/app/components/chat/window/output/output.vue

```diff
@@ -199,10 +215,40 @@ export default class AppChatWindowOutput extends Vue {
 			return;
 		}
 
-		if (el.scrollHeight - (el.scrollTop + el.offsetHeight) > 10) {
+		const getOffsetFromBottom = () => {
```

Comment:
i mean, at this point, isn't it just faster to type `el.` before each variable? haha